docs/development/release_management.rst

```diff
@@ -127,23 +128,32 @@ Pre-Release
 #. Build Debian packages:
 
    a. Check out the tag for the release candidate.
-   #. Build the packages with ``make build-debs``.
-   #. Build logs should be saved and published according to the `build
+   b. Build the packages with ``make build-debs``.
+
+     .. note:: Since we don't update the container that builds the debian packages at runtime, you
+               might end up seeing ``test_ensure_no_updates_avail`` fail in the build output. This
+               tells us that our builder image needs to be updated to ensure we are using the latest
+               security packages. You can ignore this test failure for release candidates, but
+               create a ticket in the ``securedrop`` repository so that we can update the builder
+               image before building the packages for the final release.
```

Comment:
It it necessary to create an issue to update the builder image (historically it doesn't look like we've done so), or is it standard operating procedure to always update it before building the packages for the final release?

Reply:
I think you're right. Perhaps creating an issue to update the builder image is overkill. We could just add in the "Release Process" section below a step for updating the builder image if this test fails. So the note here could say something more like:
```
     .. note:: Since we don't update the container that builds the debian packages at runtime, you
               might end up seeing ``test_ensure_no_updates_avail`` fail in the build output. This
               tells us that our builder image needs to be updated to ensure we are using the latest
               security packages. You can ignore this test failure for release candidates, because
               it will be updated as part of the final release (linking to the release process section).
```

Reply:
That sounds right to me -- @emkll, does that make sense to you as well?

Reply:
Yes, @creviera that language looks good to me. We could also link to the build instructions, since the release management documentation is generally developer-facing: docs/development/dockerbuildmaint.rst


Note that in the "Final Release" section of Release issues, we have a check box in place to ensure the builder is up-to-date, so i agree that opening an issue may be redundant here:
```
 Ensure builder in release branch is updated and/or update builder image
```

Reply:
okay i also update docs/development/dockerbuildmaint.rst so this is ready again for review